src/Web/Sfa.Das.Sas.ApplicationServices/Models/ProviderSearchResults.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Sfa.Das.Sas.ApplicationServices.Models
 {
-    public sealed class ProviderSearchResults : BaseProviderSearchResults
+    public class ProviderSearchResults : BaseProviderSearchResults
```

Comment:
think this shouldnt be plural as its as single result

Reply:
It has a property: `IEnumerable<ProviderSearchResultItem> Hits` which has the results.